### ISSUE TITLE
请教环境设置

### ISSUE THREAD

**Issue by zhaoyuanmachina** [thunlp/explore-and-evaluate#2]
暂时环境配为python3.7.0，pytorch=1.2.0，scikit-learn=0.23.2，transformers=3.4.0，pytorch_transformers=1.1.0
报段错误
请问作者具体的运行环境

**Comment by zhaoyuanmachina** [comment]
第一次运行之后的结果如下
(py37) guest20@SR650:~/tuantuan/Attr_GCN/explore-and-evaluate-main$ python train_subgraph.py --gpu_id 0 --channel Literal --dataset DBP15k/zh_en --load_hard_split
[2020-12-17 15:25:55] Loading adversarially-splitted train/valid/test set from bin/DBP15k/zh_en/hard_split.
100%|█████████████████████████████████| 213450/213450 [00:09<00:00, 23472.16B/s]
100%|██████████████████████████████████████| 433/433 [00:00<00:00, 220244.20B/s]
100%|█████████████████████████| 435779157/435779157 [1:55:28<00:00, 62894.41B/s]
段错误 (核心已转储)

大概分析了一下，应该是这个方法出了段错误
def _split_digit_attribute_and_literal_attribute(value_and_attribute_pairs, digit_threshold, att_set):
    numeral = Numeral()
    att_is_number = {}
    #print(value_and_attribute_pairs)
    for value, att_id in value_and_attribute_pairs:
        # 0 digit, 1 literal
        is_numeral, number = numeral.is_numeral(value)
        if att_id in att_is_number:
            if is_numeral:
                att_is_number[att_id][0] += 1
            else:
                att_is_number[att_id][1] += 1
        else:
            if is_numeral:
                att_is_number[att_id] = [1, 0]
            else:
                att_is_number[att_id] = [0, 1]

    digit_atts = {att for att, count in att_is_number.items() if count[0] / sum(count) > digit_threshold}
    literal_atts = {att for att in att_set if att not in digit_atts}
    digit_atts = list(digit_atts)
    digit_atts.sort()
    literal_atts = list(literal_atts)
    literal_atts.sort()
    digit_att2id = {digit_att: idx for idx, digit_att in enumerate(digit_atts)}
    literal_att2id = {literal_att: idx for idx, literal_att in enumerate(literal_atts)}
    return digit_att2id, literal_att2id

求指点

**Comment by zhaoyuanmachina** [comment]
在torch==1.6.0
torchvision==0.7.0
transformers==2.1.1
pytorch-transformers==1.1.0可运行
